src/ParameterLists.f90

```diff
@@ -2688,13 +2726,39 @@ SUBROUTINE verify_Paramtype(thisParam,reqParams,isMatch)
       !Assume the list is valid, check it only if the required parameter
       !list is not empty.
       isValid=.TRUE.
-      isMatch=.FALSE.
+      isMatch=.TRUE.
       IF(ASSOCIATED(reqParams%pdat)) THEN
-        isValid=validateReq_ParamType(thisParam,reqParams,'',isMatch)
+        isValid=validateReq_ParamType(thisParam,reqParams,'',VALIDTYPE_VERIFYTEST,isMatch)
       ELSE
         isMatch=.NOT.ASSOCIATED(thisParam%pdat)
       ENDIF
-    ENDSUBROUTINE verify_Paramtype
+    ENDSUBROUTINE verifyTest_Paramtype
+!
+!-------------------------------------------------------------------------------
+!> @brief Verify should only be used in a unit test setting.  It is for checking
```

Comment:
Again split this comment.

Reply:
Clarify if this routine checks for consistent entries or consistent entries _and_ values of data for each entry

Reply:
Roger.

Reply:
Updated.